src/app/pages/credentials/new-users/all-users/user-details/user-profile-card/user-profile-card.component.html

```diff
@@ -0,0 +1,37 @@
+<mat-card class="card">
+  <mat-card-header>
+    <h3 mat-card-title>
+      {{ 'Profile' | translate }}
+    </h3>
+  </mat-card-header>
+
+  <mat-card-content>
+    @if (user(); as user) {
+      <p class="row">
+        <span class="label">{{ 'Full Name' | translate }}:</span>
+        <span class="value">{{ user.full_name }}</span>
+      </p>
+      <p class="row">
+        <span class="label">{{ 'Email' | translate }}:</span>
+        <span class="value">{{ user.email || 'N/A' | translate }}</span>
+      </p>
+      <p class="row">
+        <span class="label">{{ 'Group' | translate }}:</span>
+        <span class="value">{{ user.group.bsdgrp_group }}</span>
+      </p>
+      <p class="row">
+        <span class="label">{{ 'Type' | translate }}:</span>
+        <span class="value">{{ user.local ? ('Local' | translate) : ('Remote' | translate) }}</span>
```

Comment:
Add a bit of logic here. 
If built-in, show "Built-In".
then If local, show "Local"
else show "Directory Services"